yarn.lock

```diff
@@ -1,4430 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
yarn lockfiles should be committed https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/